.github/workflows/doxygen.yml

```diff
@@ -35,11 +35,10 @@ jobs:
         # If you're changing the branch from main,
         # also change the `main` in `refs/heads/main`
         # below accordingly.
-        # /!\ tmp 
         if: github.ref == 'refs/heads/develop'
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           publish_dir: ./html
-
+          destination_dir: ./doxygen
```

Comment:
Is the `if:` at line 38 really required ? This action is only triggered on a push to branch `develop`

Reply:
right!